contexts/zcap-v1.jsonld

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "allowedAction": "https://w3id.org/security#allowedAction",
+    "capability": {"@id": "https://w3id.org/security#capability", "@type": "@id"},
```

Comment:
We shouldn't do anything about this now ... because it's not clear what should be done, but if this property is intended to be used with values that are capabilities, we may want the container type to be a graph. That being said, I remember we didn't do this originally for some reason -- and I'm not sure if it was for backwards compatibility or what. I'd love to have more feedback from others on what should be done here. There may be similar concerns around `capabilityChain`. I'll file an issue.

Reply:
Thanks!